src/assets/translations/en/main.json

```diff
@@ -2872,7 +2872,8 @@
     },
 
     "cta": {
-      "title": "You have TCY claims available!",
+      "title": "Have any THORChain savers or lending positions?",
```

Comment:
This feels a bit weird - if we are rendering this then we _know_ they don't have any TCY claims available.

Should we instead prompt them to try another wallet?

Reply:
Agree translation feels weird here - would be in favour of keeping this much more generic like "THORChain TCY claims have landed!" and then let users handle whichever account they wish to in TCY page.
There's an empty state there to prompt to try with another wallet:

<img width="1407" alt="image" src="https://github.com/user-attachments/assets/11b6cf44-8915-4590-aadf-298bd26cb1d6" />